installers/npm/binary.js

```diff
@@ -212,13 +212,19 @@ class Binary {
   }
 }
 
-const getBinary = () => {
+const getBinary = (overrideInstallDirectory = "") => {
```

Comment:
nit: this could probably just be `overrideInstallDirectory` and let it evaluate to `undefined` if no one passes a value for it